test/objectmancy/objectable/mass_update_test.rb

```diff
@@ -26,4 +26,10 @@ def test_properly_calls_any_custom_methods
 
     assert_equal('smile', test_obj.video_game.developer)
   end
+
+  def test_method_returns_self
+    test_obj = ObjTestClasses::TestObject.new(name: 'old name')
+    test_obj.mass_update(name: 'new', primes: [1, 2])
+    assert_equal(test_obj, test_obj.mass_update(name: 'new', primes: [1, 2]))
```

Comment:
I think this should be an additional assertion of `assert_same`, since that would use `Object#equal?`, verifying they have the same object identity. `assert_equal` is defined using the `==` method, which is given in the module. We don't care if they're equivalent in terms of values; we want to be sure it returns the same instance. There are already tests that verify we can update values.